client/app/components/packages/pas-form/edit.hbs

```diff
@@ -704,6 +704,22 @@
             {{!-- TODO: Make this button work with the action passed in @save--}}
             Save
           </button>
+          {{!-- Error Handling --}}
+          {{#if this.saveError}}
+            <div class="callout alert" data-test-error-saving>
+                <h5 class="text-red-dark">
+                  <FaIcon
+                    @icon="exclamation-circle"
+                    @size="2x"
+                    class="text-red float-left medium-margin-right tiny-margin-top"
+                  />
+                  There was a problem saving your information.
+                </h5>
+                <p class="text-red-dark text-small">
+                  Please try again. If the problem persists, please contact City Planning at <a href="mailto:ZAP_feedback_DL@planning.nyc.gov">ZAP_feedback_DL@planning.nyc.gov</a> or <a href="tel:212-720-3300" class="nowrap">212-720-3300</a>.
+                </p>
+            </div>
+          {{/if}}
```

Comment:
Shouldn't we also display the error message?

There's an API for dealing with errors on Ember Data models: https://davidtang.io/2016/01/09/handling-errors-with-ember-data.html

In fact, you could get away with just using template logic.

Take a look at `projects-error.hbs` to see some examples... 

Also, do you think we could put these error messages  messages into a component? Take a look at the length of edit.hbs... it's already pretty long!

Gimme call if you wanna talk through approach! :)

Reply:
Chatted with @andycochran and we didn't think it needed to be separated out into a separate component since there were only two instances where we used it, but I agree that edit.hbs is getting pretty long! Separating out into a component shouldn't be too hard if we want to shorten this file.

In response to "Shouldn't we also display the error message" -- how will the error message help our users? If you think it's important for them to know whether it's a 404, a 500, or some other error then yes we can add that, but I feel like that could get messy/might not be super relevant to our users? Let me know if you think differently

Reply:
Yeah, but this component is too long already, don't you think? We don't want to put ourselves in a position where it's so hard to maintain because it's over 1000 lines long.

As for showing the error message — including the status code — it will help our users provide better feedback to us so that we can fix whatever is causing the error in the first place. Usually they provide a screenshot. I think this is a cheaper way of getting bugs fixed faster, based on our experience with the LUP portal (where we didn't provide that feedback).

Great questions!

Reply:
Good to know, will do!